client/src/components/App/App.css

```diff
@@ -29,7 +29,9 @@
   --font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
 }
 
-*, *::before, *::after {
```

Comment:
Is there anything to call out in this file? I believe these are all auto-formatting changes

Reply:
Yes, this file got auto formatted. there are no changes in this